components/Newsletter.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { useState } from 'react'
+import classnames from 'classnames'
+
+interface NewsletterFormProps {
+  wrapperClassName?: string
+  className?: string
+}
+const NewsletterForm: React.FC<NewsletterFormProps> = ({ wrapperClassName = '', className = '' }) => { 
+  const [email, setEmail] = useState('')
+  const [username, setUsername] = useState('')
+
+  // const handleSubmit = (e: any) => {
+  //   e.preventDefault()
+  //   // Handle form submission here, such as sending a request to subscribe the user
+  // }
+
+  // const handleFileChange = (e: any) => {
+    
+  // }
+
+  return (
+    <section
```

Comment:
We are very close with the desktop implementation, however we need to keep working for mobile and small devices displays.

![Screenshot 2024-02-21 at 11 39 47](https://github.com/json-schema-org/website/assets/40007659/f4aa8b04-e6b5-4ad8-b5d6-617deaa1b884)

Please see the behaviour of Asyncapi component and adjust accordingly
![Screenshot 2024-02-21 at 11 39 35](https://github.com/json-schema-org/website/assets/40007659/588260ae-02d9-4178-977c-627170bafc7f)
: